docker/release/dockerfiles/opensearch.al2.dockerfile

```diff
@@ -104,7 +105,8 @@ LABEL org.label-schema.schema-version="1.0" \
   org.label-schema.license="Apache-2.0" \
   org.label-schema.vendor="OpenSearch" \
   org.label-schema.description="$NOTES" \
-  org.label-schema.build-date="$BUILD_DATE"
+  org.label-schema.build-date="$BUILD_DATE"\
```

Comment:
Needs a space before `\`?

Reply:
Ooh yeah, i am sorry
Must have missed that.Fixed that.